frontend/src/v5/ui/routes/viewer/tickets/ticket.context.tsx

```diff
@@ -23,22 +23,22 @@ export enum TicketDetailsView {
 
 export interface TicketContextType {
 	isViewer?: boolean;
-	view: TicketDetailsView;
-	viewProps?: any;
+	detailsView: TicketDetailsView;
+	detailsViewProps?: any;
 	setDetailViewAndProps: (view: TicketDetailsView, props?: any) => void;
 }
 
 const defaultValue: TicketContextType = {
 	isViewer: false,
-	view: TicketDetailsView.Form,
+	detailsView: TicketDetailsView.Form,
 	setDetailViewAndProps: () => {},
```

Comment:
Sorry. One last thing. Should this not be `setDetailsViewAndProps` instead of `setDetailViewAndProps`

Reply:
there are two fields that are set by that function `detailsView` and `detailsViewProps`

Reply:
Yes, so why is it detail and not details in this case?

Reply:
Ahh ok. cool. I'll rename that in another PR, is not worth to get another round of reviews just for an s.